presets/shared.yml

```diff
@@ -71,6 +71,7 @@ symbolWalletImage: symbolplatform/symbol-desktop-wallet:1.0.1
 symbolFaucetImage: symbolplatform/symbol-faucet:1.0.1-alpha
 symbolAgentImage: symbolplatform/symbol-node-rewards-agent:2.0.0
 mongoImage: mongo:4.4.3-bionic
+httpsPortalImage: steveltn/https-portal:1
```

Comment:
we would need an alpha rest eventually to test the native implementation

Reply:
Yes, I'm currently working on the `catapult-rest` part.

Reply:
https://github.com/symbol/catapult-rest/pull/641